### DISCUSSION TITLE
can't open GUI file explorer at top-level directory

### DISCUSSION THREAD

**Discussion by ultraGentle** [community/community#8966] [Codespaces] [answered]
When you start a Codespace, it puts you in `/workspaces/repoName`, both in the terminal and the sidebar file explorer.  

From the terminal, you can `cd /` into the top level directory, and do command-line stuff.  

But the file explorer can't seem to navigate "up" directories past `workspaces/repoName`.  

From desktop VSCode, you can connect to a GitHub Codespace and `cd / && code .` to open a new editor in the top level directory, but it's clunky and opens a separate editor. 

Is there a way to set the GUI file explorer to open at the top level directory?

**Comment by dmalan** [comment, marked as answer]
Hi @ultraGentle, in Codespaces, you should be able to select **File > Open Folder...**, then `/`. Or you can append `/?folder=%2F` to your `github.dev` URL, which should also open `/` in the explorer!

**Reply by andrecastropaiva** [reply]
Your answer helped me as well David. Thanks

**Reply by ababyprogrammer** [reply]
WOW! Soo help and usefull. Thanks **Mr.Malan**! ( $ _ $ )

**Comment by sammonius** [comment]
I found that if you right-click the explorer and press "Add folder to workspace", you can type the path to your directory and it will appear on the explorer.

**Comment by JewelArimattom** [comment]
Add ?folder=/ to the end of the Codespace URI to open it in the root directory. This makes the URI look like https://username-repo-random.github.dev/?folder=/.

Alternatively, you can use the integrated terminal and run cd / && code . from there to open a new tab with the Codespace in the root directory. The opened Codespace will have ?folder=/ in the URI, so this is effectively equivalent to the other way I described.

**Reply by JewelArimattom** [reply]
helps me

**Reply by cyber-250** [reply]
help full

**Reply by good-gtt** [reply]
Your answer helped me
